config/clusters/2i2c/ohw.values.yaml

```diff
@@ -1,3 +1,7 @@
+dask-gateway:
+  gateway:
```

Comment:
I think you want to set nodeSelector for both scheduler and worker config, but not gateway - gateway is a 'core' node and so should just stay in the core node pool.

Reply:
Hmm, I see this is perhaps newer dask-gateway config! From https://github.com/dask/dask-gateway/blob/37c4ead0f0c91366d6eea39ee47b6d1cc0ae5793/resources/helm/dask-gateway/values.yaml#L140 I can see it says 'the gateway pods' but it's not clear which pods exactly these are. The worker and scheduler pods come and go so they should be on the ohw specific nodes, but the controller and traefik pods stay on forever and so should be on the core nodes. 

I *think* this config as it is is correct, and does the right thing! So actually ignore everything I said haha.